lib/waylon/jenkins/job/rest.rb

```diff
@@ -70,7 +74,7 @@ def investigating?
         end
 
         def description
-          @job_build_description ||= @client.job.get_build_details(@name, last_build_num)['description']
+          @job_build_description ||= job_build_details['description']
```

Comment:
Some ideas, in maybe increasing order of effort & "right"ness:

 - Check for `nil` before de-referencing the key
 - Create wrapper methods for the `job_build_details` fields of interest
 - elevate the `job_build_details` instance to a proper class instead of a `Hash`